src/qs_localization_methods.F

```diff
@@ -2208,6 +2220,13 @@ SUBROUTINE jacobi_rot_para(weights, zij, vectors, para_env, max_iter, eps_locali
 
       CALL cp_fm_get_info(rmat, nrow_global=nstate)
 
+      IF (restricted > 0) THEN
+         IF (output_unit > 0) THEN
+            WRITE (6, *) "JACOBI: The last ", restricted, " orbitals do not rotate"
```

Comment:
Same, see [here](https://www.cp2k.org/conv#c012).

Reply:
Sorry, I missed it. Replaced it in the code with cp_logger_get_default_io_unit().